tools/mo/openvino/tools/mo/front/tf/RFFTRealImagToRFFTSplit.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from openvino.tools.mo.front.common.partial_infer.utils import int64_array
+from openvino.tools.mo.front.common.replacement import FrontReplacementSubgraph
+from openvino.tools.mo.front.subgraph_matcher import SubgraphMatch
+from openvino.tools.mo.front.tf.graph_utils import create_op_with_const_inputs
+from openvino.tools.mo.graph.graph import Graph, rename_nodes
+from openvino.tools.mo.ops.split import Split
+from openvino.tools.mo.ops.squeeze import Squeeze
+
+
+class RFFTRealImagToRDFTSplit(FrontReplacementSubgraph):
+    """
+    This transformation converts the operation TFRFFT into OpenVINO RDFT.
+    """
+    enabled = True
+
+    def run_before(self):
+        from openvino.tools.mo.front.tf.TFFFTToDFT import TFFFTToDFT
+        return [TFFFTToDFT]
+
+    def pattern(self):
+        return dict(
+            nodes=[
+                ('rfft', dict(op='TFFFT', fft_kind='RDFT')),
+                ('real', dict(op='Real')),
+                ('imag', dict(op='Imag')),
+            ],
+            edges=[
+                ('rfft', 'real', {'in': 0}),
+                ('rfft', 'imag', {'in': 0}),
+            ])
```

Comment:
I think we can do more universal transformation from this to handle cases like: `RDTF->Real`, `RDFT->Imag`, and both.
I can avoid pattern and walk through all `Real` and `Imag` nodes separately which have `RDFT` as a producer and replace them with corresponding `Split` nodes.

Reply:
Moreover, this transformation looks the same as `RollRealImagPack` expect axis adjustment. We can combine these two transformations.